src/Sylius/Bundle/ApiBundle/Resources/config/serialization/Order.xml

```diff
@@ -105,6 +105,7 @@
         </attribute>
 
         <attribute name="itemsTotal">
+            <group>admin:order:read</group>
```

Comment:
you've changed serialization so you should update/create contract tests for order (I guess).